packages/utils/types/feed.ts

```diff
@@ -135,3 +136,19 @@ export type ReplyToType = {
   yOffsetValue?: number;
   parentId?: string;
 };
+
+export const parseSocialFeedMetadata = (
+  category: PostCategory,
+  metadata: string,
+) => {
+  switch (category) {
+    case PostCategory.Video:
+      return zodTryParseJSON(ZodSocialFeedVideoMetadata, metadata);
+    case PostCategory.Article:
+      return zodTryParseJSON(ZodSocialFeedPostMetadata, metadata);
+    case PostCategory.MusicAudio:
+      return zodTryParseJSON(ZodSocialFeedTrackMetadata, metadata);
+    default:
+      return zodTryParseJSON(ZodSocialFeedPostMetadata, metadata);
+  }
```

Comment:
```suggestion
  switch (category) {
    case PostCategory.Video:
      return zodTryParseJSON(ZodSocialFeedPostMetadata, metadata) || zodTryParseJSON(ZodSocialFeedVideoMetadata, metadata);
    case PostCategory.Article:
      return zodTryParseJSON(ZodSocialFeedPostMetadata, metadata) || zodTryParseJSON(ZodSocialFeedArticleMetadata, metadata);
    case PostCategory.MusicAudio:
      return zodTryParseJSON(ZodSocialFeedTrackMetadata, metadata);
    default:
      return zodTryParseJSON(ZodSocialFeedPostMetadata, metadata);
  }
```
- Use ZodSocialFeedArticleMetadata for Articles
- Hanlde the "old" posts. Example: https://github.com/TERITORI/teritori-dapp/blob/bugfix/video-audio-map/packages/components/socialFeed/Map/MapPosts/ArticleMapPost.tsx#L24-L26

We'll make another PR to replace all `zodTryParseJSON` usages by  `parseSocialFeedMetadata` (About posts)

Reply:
> Hanlde the "old" posts

actually we will reset the feed after redesigning the contracts so I don't think it's important to support old stuff right now, even all this refacto is not necessary since we'll have well defined types from contracts

Reply:
I made changes here: https://github.com/TERITORI/teritori-dapp/pull/1400/commits/8a0ebc639cf8ad35aee3d70b4fa098f3a0bfd133

Reply:
@n0izn0iz Yes, some issues are "stand by" for now